nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/RunNiFi.java

```diff
@@ -250,18 +252,13 @@ public static void main(String[] args) throws IOException, InterruptedException
                 }
                 dumpFile = new File(args[2]);
             } else {
-                try {
-                    Paths.get(args[1]);
-                } catch (InvalidPathException e) {
-                    System.err.println("Invalid filename. The command parameters are: status-history <number of days> <dumpFile>");
+                final boolean isValid = DumpFileValidator.validate(args[1]);
```

Comment:
Would not it make sense to use this validation at L248 too, instead of calling `Paths.get(args[2])`?

Reply:
Yes, I missed that. Thanks!